assignments/callbacks.js

```diff
@@ -41,25 +41,71 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  console.log(cb(arr))
+
+}
+function itemlength(arr) {
+  return arr.length;
 }
 
+getLength(items, itemlength); // end 1
+
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  console.log(cb(arr))
+}
+
+function lastitem(arr){
+for(var i = 0; i < arr.length; i++){
+  return arr[3]
```

Comment:
this is a hard coded value. what would happen if additional things were added to the array would this really return the last item? what is another way to get the last item without hardcoding it?